Grid/qcd/action/gauge/GaugeImplementations.h

```diff
@@ -193,6 +193,11 @@ typedef ConjugateGaugeImpl<GimplTypesR> ConjugateGimplR; // Real.. whichever pre
 typedef ConjugateGaugeImpl<GimplTypesF> ConjugateGimplF; // Float
 typedef ConjugateGaugeImpl<GimplTypesD> ConjugateGimplD; // Double
 
+typedef PeriodicGaugeImpl<SpGimplTypesR> SpPeriodicGimplR; // Real.. whichever prec
+typedef PeriodicGaugeImpl<SpGimplTypesF> SpPeriodicGimplF; // Float
+typedef PeriodicGaugeImpl<SpGimplTypesD> SpPeriodicGimplD; // Double
+
```

Comment:
As for the fermions, I don't know what they are needed for. So I wouldn't add them

Reply:
Okay. Maybe we should start taking notes on what to ask Peter about in the PR's comment. This would be one of them (in case he doesn't stumble across it himself in review).